fossunited/chapters/web_template/foss_community_cta/foss_community_cta.html

```diff
@@ -0,0 +1,112 @@
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@tabler/icons-webfont@2.46.0/tabler-icons.min.css">
+
+<style>
+
+    .cta-section{
+        display: flex;
+        font-family: Inter;
+        background: #F5F5F5;
+
+    }
+
+    .cta-container{
+        display: flex;
+        gap: 3rem;
+        padding: 5rem 10rem 0rem 10rem;
+    }
+
+    .cta-content{
+        display: flex;
+        flex-direction: column;
+        padding-bottom: 5rem;
+        gap: 2rem;
+    }
+
+    .cta-text{
+        display: flex;
+        flex-direction: column;
+        gap: 0.5rem;
+    }
+
+    .cta-text h2{
+        margin: 0;
+        color: var(--Foundation-Code-Night-code-night-500, #1A1A1A);
+        font-size: 1.75rem;
+        font-weight: 600;
+        line-height: 100%; /* 1.75rem */
+        letter-spacing: -0.105rem;
+    }
+
+    .cta-text p{
+        color: var(--Gray-500, #667085);
+        font-size: 1.125rem;
+        font-weight: 400;
+        line-height: 1.75rem;
+        letter-spacing: -0.105rem;
+    }
+
+    #cta-button{
+        width: 50%;
+    }
+
+    .cta-image{
+        display: flex;
+        align-items: flex-end;
+    }
```

Comment:
Maybe add a cta-image style for mobile view also. It is too big right now

Reply:
Done - reduced image size by 20% in mobile view